tip-router-operator-cli/tests/integration_tests.rs

```diff
@@ -201,108 +201,108 @@ impl TestContext {
 }
 
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
-async fn test_process_epoch() -> Result<(), Box<dyn std::error::Error>> {
-    let mut test_context = TestContext::new().await?;
-
-    // Create required directories
-    let ledger_path = test_context.temp_dir.path().join("ledger");
-    fs::create_dir_all(&ledger_path)?;
-    fs::create_dir_all(&test_context.output_dir)?;
-
-    // Create test CLI struct
-    let cli = Cli {
-        keypair_path: test_context.temp_dir
-            .path()
-            .join("keypair.json")
-            .to_str()
-            .unwrap()
-            .to_string(),
-        rpc_url: "http://mock".to_string(),
-        ledger_path: test_context.temp_dir.path().join("ledger"),
-        snapshot_output_dir: test_context.output_dir.clone(),
-        command: Commands::Monitor {
-            ncn_address: Pubkey::new_unique(),
-            tip_distribution_program_id: TIP_DISTRIBUTION_ID,
-            tip_payment_program_id: TIP_PAYMENT_ID,
-        },
-    };
-
-    // Write keypair file
-    fs::write(&cli.keypair_path, test_context.payer.to_bytes().to_vec())?;
-
-    // let test_client = TestEllipsisClient {
-    //     banks_client: test_context.context.banks_client.clone(),
-    //     payer: test_context.payer.clone(),
-    //     rpc_client: RpcClient::new_mock_with_mocks("mock".to_string(), vec![]),
-    // };
-
-    let snapshot_creator = MockSnapshotCreator::new(
-        test_context.context.banks_client.clone(),
-        test_context.output_dir.clone()
-    );
-    let result = process_epoch(
-        64,
-        &cli,
-        &test_context.payer,
-        snapshot_creator,
-        &TIP_DISTRIBUTION_ID,
-        &TIP_PAYMENT_ID,
-        &Pubkey::new_unique()
-    ).await;
-
-    match result {
-        Ok(_) => {
-            // Verify snapshot directory was created
-            let snapshot_dir = cli.snapshot_output_dir.join("snapshot");
-            assert!(snapshot_dir.exists(), "Snapshot directory was not created");
+// #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+// async fn test_process_epoch() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Lets delete this entire test instead of commenting it out. 

A) No need to keep it given the new game plan
B) We can always find old code in the commit history, so commenting it out doesn't add value to codebase in current state

Reply:
roger that